packages/manager/apps/dedicated/client/app/dedicated/server/details/server.controller.js

```diff
@@ -664,4 +667,19 @@ export default class ServerCtrl {
       `#/nutanix/${clusterServiceName}/nodes/${nodeServiceName}`,
     );
   }
+
+  fetchRTMInfo() {
+    return this.Server.getRtmVersion(this.$stateParams.productId)
+      .then(() => {
+        this.Alerter.set(
+          'alert-info',
+          this.$translate.instant('server_rtm_eol_info'),
+          null,
+          'rtm_eol_info',
+        );
+      })
+      .catch(() => {
+        this.displayRTMInfo = false;
```

Comment:
where are you setting this to true?

Reply:
First, I was using this flag to display the message, later I refactored it to use `Alerter`. This is redundant, I will remove this.